From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 29 Dec 2013 23:20:30 +0100
Subject: staging: lustre: Make LUSTRE_LLITE_LLOOP tristate
Forwarded: http://mid.gmane.org/1388356095.2900.34.camel@deadeye.wl.decadent.org.uk

There seems to be no reason why it can't be a module; in fact that is
currently supposed to be the default.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 drivers/staging/lustre/lustre/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/lustre/lustre/Kconfig b/drivers/staging/lustre/lustre/Kconfig
index 2156a44..faa01fa 100644
--- a/drivers/staging/lustre/lustre/Kconfig
+++ b/drivers/staging/lustre/lustre/Kconfig
@@ -55,6 +55,6 @@ config LUSTRE_TRANSLATE_ERRNOS
 	default y
 
 config LUSTRE_LLITE_LLOOP
-	bool "Lustre virtual block device"
+	tristate "Lustre virtual block device"
 	depends on LUSTRE_FS && BLOCK
 	default m
